_posts/2016-10-06-virtual-mesh-hack-night.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title: Virtual Mesh Hack Night
+slug: virtual-mesh-hack-night
+text: Join our virtual mesh from anywhere and play with distributed services.
+location: "#virtualmesh:tomesh.net"
+locationMap: https://chat.tomesh.net/#/room/#virtualmesh:tomesh.net
```

Comment:
We aren't using the locationMap links right now.  From discussion we might to it to try and embed maps....which means it would be better to remove the URL for locationMap (and possibly the location?)